client/my-sites/stats/features/modules/stats-utm/stats-module-utm-dropdown.scss

```diff
@@ -29,7 +29,6 @@ $stats-utm-border: 1px solid var(--gray-gray-5, #dcdcde);
 
 .stats-utm-picker__popover-list {
 	padding: 8px 0;
-	border-left: $stats-utm-border;
```

Comment:
I noticed a minor styling issue: a double left border is displayed on the popover. I am including this in this revert PR. But if this is not a small fix for some reason and if there is any more feedback on this, I will remove this change get this PR merged sooner.

![Arc -2025-03-27 at 12 31 26@2x](https://github.com/user-attachments/assets/65ada9d4-c42d-420d-8044-cf6164ebe977)